source/context_menus/_context_menu_stats.js

```diff
@@ -0,0 +1,58 @@
+const { PermissionFlagBits, InteractionContextType } = require('discord.js');
+const { UserContextMenuWrapper } = require('../classes');
+
+const mainId = "User Stats";
+module.exports = new UserContextMenuWrapper(mainId, null, false, [ InteractionContextType.Guild ], 3000,
+	/** Specs */
+	(interaction, database, runMode) => {
+		const target = interaction.targetMember;
+		if (target.id == interaction.client.user.id) {
+			// BountyBot
+			buildCompanyStatsEmbed(interaction.guild, database).then(embed => {
+				interaction.reply({
+					embeds: [embed],
+					ephemeral: true
+				});
+			})
+		} else {
+			// Other Hunter
+			database.models.Hunter.findOne({ where: { userId: target.id, companyId: interaction.guildId } }).then(async hunter => {
+				if (!hunter) {
+					interaction.reply({ content: "The specified user doesn't seem to have a profile with this server's BountyBot yet. It'll be created when they gain XP.", ephemeral: true });
+					return;
+				}
+
+				const { xpCoefficient } = await database.models.Company.findByPk(interaction.guildId);
+				const currentLevelThreshold = Hunter.xpThreshold(hunter.level, xpCoefficient);
```

Comment:
Got a crash on `Hunter` not being imported here (for it trying to use the static method).

Reply:
That...shouldn't be the case? If we imported hunter here, it shouldn't matter, since we're exclusively accessing Hunter via the database model list, and the imported Hunter object (not fed into the Sequelize functionality) likely wouldn't have (and shouldn't have) that function in the first place, at least at the line that I *think* you are pointing at. If you are pointing to the ones below this, that isn't clear by how you've placed this comment.

Reply:
It's the `Hunter.xpThreshold()` static method that's the issue. I agree it's weird that github seems to know Hunter's a class, but the crash was found while running the command on the branch.